fastlane/lib/fastlane/actions/restore_file.rb

```diff
@@ -5,7 +5,7 @@ def self.run(params)
         path = params[:path]
         backup_path = "#{path}.back"
         UI.user_error!("Could not find file '#{backup_path}'") unless File.exist? backup_path
-        FileUtils.cp(backup_path, path, {preserve: true})
+        FileUtils.cp(backup_path, path, { preserve: true })
```

Comment:
in contrast to the one above, this was corrected to having leading and trailing spaces. 👍